Routing/Loader/RestRouteLoader.php

```diff
@@ -126,6 +126,10 @@ private function getControllerLocator($controller)
             if ($useScope) {
                 $this->container->leaveScope('request');
             }
+
+            if ($controller === $class && class_exists($controller)) {
```

Comment:
I don't think we need the class existence check as $class comes from php itself.